report/finished/chapters/modeling.tex

```diff
@@ -1,6 +1,206 @@
-\section*{Robotic arm}
-\subsection*{Forward kinematics}
-\subsection*{Inverse kinematics}
+
+%%%%%%%%%%%%%%%% arm kinematics %%%%%%%%%%%%%%
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.6\textwidth]{chapters/img/robot_arm.png}
+    \caption{Placeholder image for the robotic arm?.}
+    \label{fig:robotic_manipulator_general}
+\end{figure}
+
+
+\section*{Manipulator Kinematics} %Niklas
+Kinematics is what describes the motion of rigid bodies and points in space. 
+
+\subsection*{Backgroud}
+\subsection*{Rigid body transformation}
+In general any position can be described by translation along 3 axes and rotation along these same axes. These translations/rotations can be described by a matrix
```

Comment:
axis

Reply:
write three instead 3.

Reply:
I think axes is correct since it is the plural form of axis. What do you think? @mnsnor